### ISSUE TITLE
Add a new regular field

### ISSUE THREAD

**Issue by evbocharov** [angryip/ipscan#350]
I have text file with list hostnames.   
![example](https://user-images.githubusercontent.com/103121081/161982755-b181d413-7be2-476f-8c0d-40cd983011f8.JPG)
Put these file in ipscan 
[list.txt](https://github.com/angryip/ipscan/files/8427213/list.txt)
and scan - we see hostname (these hostname is loading from internet)
![scan](https://user-images.githubusercontent.com/103121081/161985009-ab644499-895c-44a9-8705-f60612326087.JPG)

But i can see a name from list.txt (not ip addresses)
Name1: google.com
Name2: yahoo.com
Name3: aol.com
Can you add a a new regular field - may be named "NAME"
These  field "NAME" will loaded from a txt file:
Example a txt file:
IP                                 NAME                                                    
google.com                 google.com
yahoo.com                  yahoo.com
aol.com                       aol.com

Thank you

**Comment by angryziber** [comment]
Do you have a use case for that?
All these names resolve to hundreds of IPs and ipscan shows you the real names of these IPs already

**Comment by evbocharov** [comment]
Yes, i use. Have list ip's (70 hosts) and i ping from internet. 
not every day, but i must check work or not servers. And important check a ping (ms) who is lower
